### PR TITLE
LPS-98040 Show comment/cancel buttons

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by victorg1991** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by victorg1991** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/ConnectedComponent.es.js [hunk 1]
```diff
@@ -95,45 +96,41 @@ const getConnectedComponent = (Component, properties) => {
...
+			}, [store]);
+
+			return store ? (
+				<WrappedComponent
+					{...props}
+					{...mapStateToProps(storeState, props)}
+					{...mapDispatchToProps(store.dispatch, props)}
+				/>
+			) : (
+				<></>
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/ConnectedComponent.es.js]: This is a React fragment shorthand?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/sidebar/comments/FragmentComments.es.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	const newCommentId = `${props.portletNamespace}newCommentId`;
+
+	return (
+		<div
+			data-fragments-editor-item-id={props.fragmentEntryLinkId}
+			data-fragments-editor-item-type={
+				FRAGMENTS_EDITOR_ITEM_TYPES.fragment
+			}
+		>
+			<dt class='mb-2 sidebar-dt text-secondary'>
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/sidebar/comments/FragmentComments.es.js]: Why using a dt here?

**Comment by victorg1991** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/75659. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by victorg1991** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/75685. [See changes here.](https://github.com/victorg1991/liferay-portal/compare/00c4c35cab36~1...00c4c35cab36) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/ConnectedComponent.es.js (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/ConnectedComponent.es.js]: Oui

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/sidebar/comments/FragmentComments.es.js (hunk 2 above)
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/sidebar/comments/FragmentComments.es.js]: no clue, my bad

**Pull request closed without merging**